x/proof/types/types.go

```diff
@@ -1,3 +1,58 @@
 package types
 
-// This file is in place to declare the package for dynamically generated protobufs
+import (
+	storetypes "cosmossdk.io/store/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+)
+
+// ClaimsIterator provides iteration functionality over claims stored in the blockchain state.
+// It wraps the underlying store iterator, primary store, and codec to provide convenient access to Claim objects.
+// The iterator uses a two-level storage approach where the main iterator provides keys that reference the actual
+// claim data in the primary store.
+type ClaimsIterator struct {
```

Comment:
Does this need to be publically exposed?

Reply:
Yes, it is the return type of [`ProofKeeper#GetSessionEndHeightClaimsIterator`](https://github.com/pokt-network/poktroll/blob/9d6317ca2dad6de0f739f0ec34d0c8fa337a2573/x/proof/keeper/claim.go#L79)